check_upgrade.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+VERSION=3.1.0.1971
+
+# stop current cluster
+./start-splice-cluster -k
+
+# clean up platform_it
+cd platform_it
```

Comment:
(optional) you also assume you're in the base directory of the project which is not necessarily true, e.g. I could this script from `/tmp`.

Maybe you could add some sanity checks that you're in a git repository and then go to the base folder using `git rev-parse --show-cdup`